files/en-us/web/svg/tutorial/clipping_and_masking/index.html

```diff
@@ -82,6 +82,6 @@ <h3 id="Transparency_with_opacity">Transparency with <code>opacity</code></h3>
 
 <h2 id="Using_well-known_CSS_techniques">Using well-known CSS techniques</h2>
 
-<p>One of the most powerful tools in a web developer's toolbox is <code>display: none</code>. It is therefore little surprise, that it was decided to take this CSS property into SVG as well, together with <code>visibility</code> and <code>clip</code> as defined by CSS 2. For reverting a previously set <code>display: none</code> it is important to know, that the initial value for all SVG elements is <code>inline</code>.</p>
+<p>One of the most powerful tools in a web developer's toolbox is <code>display: none</code>. It is therefore a little surprising, that it was decided to take this CSS property into SVG as well, together with <code>visibility</code> and <code>clip</code> as defined by CSS 2. For reverting a previously set <code>display: none</code> it is important to know, that the initial value for all SVG elements is <code>inline</code>.</p>
```

Comment:
The original sentence is correct and doesn't mean the same as your fix. It means "It was not surprising". 

I propose to rephrase it like this, which simplifies the phrase.

```suggestion
<p>One of the most powerful tools in a web developer's toolbox is <code>display: none</code>. It is therefore a not a surprise that it was decided to take this CSS property into SVG as well, together with <code>visibility</code> and <code>clip</code> as defined by CSS 2. For reverting a previously set <code>display: none</code> it is important to know, that the initial value for all SVG elements is <code>inline</code>.</p>
```

Reply:
Oh you're right, my bad. I don't know how I got confused over that. Your suggestion for making it more clear is perfect.